nixos/modules/security/acme.nix

```diff
@@ -1,11 +1,296 @@
-{ config, lib, pkgs, ... }:
+{ config, lib, pkgs, options, ... }:
 with lib;
 let
-
   cfg = config.security.acme;
 
+  # Used to calculate timer accuracy for coalescing
+  numCerts = length (builtins.attrNames cfg.certs);
+  _24hSecs = 60 * 60 * 24;
+
```

Comment:
Can you add a comment here describing some basic conventions for all of these services?

They run as the acme user with their own private /tmp.
Except from the minica unit (why), they also run as the `acme` group.
Files from `/var/lib/acme` are bind-mounted into paths inside `/tmp` as-needed.

Reply:
Good idea! Wrt the minica unit's group, that's an oversight on my part. I will set it to acme.

Reply:
In that case, it can be moved into the common attrset.